action.yml

```diff
@@ -4,22 +4,21 @@ description: 'Set up a specific version of the Java JDK and add the
 author: 'GitHub'
 inputs:
   java-version:
-    description: 'The Java version to make available on the path. Takes a whole
-       or semver Java version, or 1.x syntax (e.g. 1.8 => Java 8.x).
-       Early access versions can be specified in the form of e.g. 14-ea,
-       14.0.0-ea, or 14.0.0-ea.28'
-    required: true
+    description: 'The Java version to set up. Takes a whole or semver Java version. See examples of supported syntax in README file'
+    required: false
```

Comment:
Nah! Please, require users of the `setup-java` action to commit to specific version of Java.

At least a feature release number, like: `8`, `11`,  ... `17`

Reply:
- https://github.com/actions/setup-java/pull/132#issuecomment-795918486
- https://github.com/actions/setup-java/pull/132#issuecomment-796672020

Reply:
I read and understood the comments. Yet, I don't agree with your conclusions.

I'll reply in the main thread.